src/components/streamSelect/StreamSelect.js

```diff
@@ -114,6 +127,27 @@ class StreamSelect extends Component {
             <button>Play</button>
           </form>
         </div>
+
+        <div className="latency-select-wrap">
+          <form
+            className="latency-select"
+            onSubmit={(e) => {
+              this.setCustomTargetLatency(e);
+            }}
+          >
+            <span>Latency: </span>
+            <input
+              onChange={(e) => {
+                this.updateLatencyField(e);
+              }}
+              value={this.state.customLatencyField || ""}
+              placeholder="Latency"
```

Comment:
Don't know if it fits, but I'd prefer to call this Target Latency (Input) everywhere consistently. Not to be confused with the current latency; which can differ. Or, if you'd like to keep things short, then consistently name it Latency.

Reply:
<img width="232" alt="image" src="https://user-images.githubusercontent.com/67740183/92232822-794d0680-eeaf-11ea-9077-842ceb921049.png">
What do you think about this? I'd like to keep it as short as possible as space is limited in the settings bar.

Reply:
I think it's an anti-design pattern to use placeholders like labels? Why not just use a clear label and leave the input empty? That way you can make the input smaller in width as well since it's only meant to input 1 or maybe 2 digit numbers anyhow.